### PR TITLE
Upgrade phalcon5

### PR DESCRIPTION
**Pull request by rwhite27**
Upgraded the package to phalcon5.

### CODE REVIEW

**Review by kaioken — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,12 +18,12 @@ jobs:
     name: PHP ${{ matrix.php-versions }}
     runs-on: ubuntu-latest
     env:
-      extensions: mbstring, intl, json, phalcon4, xdebug
+      extensions: mbstring, intl, json, phalcon5, xdebug
```
Comment (Jeckerson) [inline .github/workflows/tests.yml]:
```suggestion
      extensions: mbstring, intl, json, phalcon-5.0.0beta2, xdebug
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -18,12 +18,12 @@ jobs:
...
       key: cache-v0.0.2
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
+        php-versions: ["8.0"]
```
Comment (Jeckerson) [inline .github/workflows/tests.yml]:
```suggestion
        php-versions: ['7.4', '8.0']
```

File: composer.json [hunk 3]
```diff
@@ -26,18 +25,18 @@
     "forum": "https://forum.phalcon.io"
   },
   "require": {
-    "php": ">=7.2",
-    "ext-phalcon": "^4.0",
+    "php": ">=8.0",
```
Comment (Jeckerson) [inline composer.json]:
```suggestion
    "php": ">=7.4",
```

File: .github/workflows/validations.yml [hunk 4]
```diff
@@ -31,13 +31,13 @@ jobs:
     name: Static Code with PHP ${{ matrix.php-versions }}
     runs-on: ubuntu-latest
     env:
-      extensions: mbstring, intl, json, phalcon4
+      extensions: mbstring, intl, json, phalcon5
```
Comment (Jeckerson) [inline .github/workflows/validations.yml]:
```suggestion
      extensions: mbstring, intl, json, phalcon-5.0.0beta2
```

File: .github/workflows/validations.yml [hunk 5]
```diff
@@ -31,13 +31,13 @@ jobs:
...
       key: cache-v0.0.2
     needs: validate-code-style
     strategy:
       fail-fast: false
       matrix:
-        php-versions: [ '7.2', '7.3', '7.4' ]
+        php-versions: ["8.0"]
```
Comment (Jeckerson) [inline .github/workflows/validations.yml]:
```suggestion
        php-versions: ['7.4', '8.0']
```

File: .github/workflows/tests.yml (hunk 2 above)
Comment (rwhite27) [reply .github/workflows/tests.yml]: Why add php 7.4 ? @Jeckerson

Comment (rwhite27) [reply .github/workflows/tests.yml]: Why add version 7.4 ? @Jeckerson

**Comment by rwhite27** [comment]
@Jeckerson  ready to be reviewed again.

**Review by Jeckerson — APPROVED** [review]

**Pull request merged**